lib/functional_mixin.js

```diff
@@ -95,6 +108,58 @@ function busterFunctionalMixin(testCase, options)
     return this;
   };
 
+  testCase.swipeLeft = function busterFunctionalMixin_swipeLeft(target, callback)
+  {
+    // update offset value
+    var offset = this.window && this.window.innerWidth ?
+                 this.window.innerWidth * this._swipeRightOrLeftOffsetFactor :
+                 this._swipeRightOrLeftDistance;
+
+    this._swipe(target, [-Math.abs(offset), 0], callback);
+
+    return this;
+
+  };
+
+  testCase.swipeRight = function busterFunctionalMixin_swipeRight(target, callback)
+  {
+    // update offset value
+    var offset = this.window && this.window.innerWidth ?
+                 this.window.innerWidth * this._swipeRightOrLeftOffsetFactor :
+                 this._swipeRightOrLeftDistance;
+
+    this._swipe(target, [Math.abs(offset), 0], callback);
+
+    return this;
+
+  };
+
+  testCase.swipeUp = function busterFunctionalMixin_swipeUp(target, callback)
+  {
+    // update offset value
+    var offset = this.window && this.window.innerHeight ?
+                 this.window.innerHeight * this._swipeUpOrDownOffsetFactor :
+                 this._swipeUpOrDownDistance;
```

Comment:
For now I think we can stick with phones, when there is demand, we'd consider other things.